src/EnergyPlus/HeatBalFiniteDiffManager.cc

```diff
@@ -1327,83 +1321,85 @@ namespace HeatBalFiniteDiffManager {
                       "Name (or Face), Inward Material Name (or Face)");
             }
 
-            for (ThisNum = 1; ThisNum <= state.dataHeatBal->TotConstructs; ++ThisNum) {
+            for (int ThisNum = 1; ThisNum <= state.dataHeatBal->TotConstructs; ++ThisNum) {
+                auto &constructFD = state.dataHeatBalFiniteDiffMgr->ConstructFD(ThisNum);
```

Comment:
@Myoldmopar I think to be safe this shortcut should be moved down to line 1333.

Reply:
OK, I can do that in the next CppCheck branch which is already building locally now.